demo_event.c

```diff
@@ -332,8 +332,13 @@ int main(int argc, char *argv[])
 	if (signal(SIGINT, sig_handler) == SIG_ERR)
 		return -1;
 
-	pthread_mutex_init(&jbuf_lock, NULL);
+	if (argc == 2 && strcmp("need_get_data", argv[1]) == 0) {
```

Comment:
This way is usually for adding subcommand, instead of option, please following ublksrv_tgt.c for adding new option. Same with the change on demo_event.c.